x-pack/plugins/security_solution/server/lib/detection_engine/rule_response_actions/types.ts

```diff
@@ -25,3 +32,11 @@ export interface AlertWithAgent extends Alert {
 export interface ResponseActionAlerts {
   alerts: AlertWithAgent[];
 }
+
+export interface AlertsAction {
+  alert_ids: string[];
+  endpoint_ids: string[];
+  hosts: Record<string, { name: string }>;
+  parameters: Record<string, unknown>;
+  error?: string;
+}
```

Comment:
same here. Please avoid re-defining the types and use or `Pick<>` from existing ones